pkg/plugins/autodiscovery/cargo/dependencyManifest.go

```diff
@@ -0,0 +1,81 @@
+package cargo
+
+const (
+	// dependencyManifest is the Go template used to generate
+	// the manifests to update the cargo file
+	dependencyManifest string = `name: '{{ .ManifestName }}'
+sources:
+  {{ .SourceID }}:
+    name: '{{ .SourceName }}'
+    kind: 'cargopackage'
+    spec:
+      package: '{{ .DependencyName }}'
+      versionFilter:
+        kind: '{{ .SourceVersionFilterKind }}'
+        pattern: '{{ .SourceVersionFilterPattern }}'
+{{- if .WithRegistry }}
+      registry:
+        url: '{{ .RegistryURL }}'
+        rootdir: '{{ .RegistryRootDir }}'
+        auth:
+          token: '{{ .RegistryAuthToken }}'
+          headerFormat : '{{ .RegistryHeaderFormat }}'		
+{{- if .RegistrySCMID }}
+    scmid: '{{ .RegistrySCMID }}'
+{{- end }}
+{{- end }}
+  {{ .ExistingSourceID }}:
+    name: '{{ .ExistingSourceName }}'
+    kind: 'toml'
+{{- if .ScmID }}
+    scmid: '{{ .ScmID }}'
+{{- end }}
+    spec:
+      file: '{{ .File }}'
+      Key: '{{ .ExistingSourceKey }}'
+conditions:
+  {{ .ConditionID }}:
+    name: 'Ensure Cargo chart named "{{ .DependencyName }}" is specified'
+    kind: 'toml'
+{{- if .ScmID }}
+    scmid: '{{ .ScmID }}'
+{{- end }}
+    spec:
+      file: '{{ .File }}'
+      query: '{{ .ConditionQuery }}'
+    sourceid: '{{ .ExistingSourceID }}'
+targets:
+  {{ .TargetID }}:
+    name: 'Bump crate dependency "{{ .DependencyName }}" for crate "{{ .CrateName }}"'
```

Comment:
Having the version information in the title would be usefull when creating the pullrequest.
Something like 

```suggestion
    name: 'Bump crate dependency "{{ .DependencyName }}" to {{ source ".SourceID" }}"'
```

`{{ source ".SourceID" }}` must be correctly escaped as it needs to be update at run time.
In the NPM pullrequest, I generated the name from the golang code instead of the go template to make it more readable